packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx

```diff
@@ -1,71 +1,82 @@
-import React from 'react';
-import GoHome from 'react-icons/lib/go/home';
-import GoIssueOpened from 'react-icons/lib/go/issue-opened';
-import { inject, hooksObserver } from 'app/componentConnectors';
+// @ts-ignore
 import { Button } from '@codesandbox/common/lib/components/Button';
 import { dashboardUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { useOvermind } from 'app/overmind';
 import { Navigation } from 'app/pages/common/Navigation';
-// @ts-ignore
+import React, { useEffect } from 'react';
+import GoHome from 'react-icons/lib/go/home';
+import GoIssueOpened from 'react-icons/lib/go/issue-opened';
 import Dashboard from '-!svg-react-loader!@codesandbox/common/lib/icons/dashboard.svg';
-import { Sadbox } from './Sadbox';
+
 import { IFallbackComponentProps } from '../types';
 import { buildCrashReport } from './buildCrashReport';
 import {
+  Actions,
+  ButtonIcon,
   Container,
+  Content,
   Header,
   Nav,
-  Content,
-  Title,
   Subtitle,
-  Actions,
-  ButtonIcon,
+  Title,
 } from './elements';
+import { Sadbox } from './Sadbox';
 
-export const CodeSadbox = inject('store')(
-  hooksObserver(
-    ({ error, trace, store }: IFallbackComponentProps & { store: any }) => (
-      <Container>
-        <Header>
-          <Nav>
-            <Navigation title="CodeSadbox" />
-          </Nav>
-        </Header>
-        <Content>
-          <Title>Oh no! Something broke!</Title>
-          <Sadbox scale={3} />
-          <Subtitle>CodeSadbox</Subtitle>
-          <Actions>
-            {store.isLoggedIn ? (
-              <Button small secondary href={dashboardUrl()}>
-                <ButtonIcon>
-                  <Dashboard />
-                </ButtonIcon>
-                Go to Dashboard
-              </Button>
-            ) : (
-              <Button small secondary href="/">
-                <ButtonIcon>
-                  <GoHome />
-                </ButtonIcon>
-                Back to Home
-              </Button>
-            )}
-            {/*
-          // @ts-ignore */}
-            <Button
-              small
-              target="_blank"
-              rel="noopener"
-              href={buildCrashReport({ error, trace })}
-            >
+export const CodeSadbox: React.FC<IFallbackComponentProps> = ({
```

Comment:
I think using `FunctionComponent` would make the type a bit more clear.

```suggestion
export const CodeSadbox: React.FunctionComponent<IFallbackComponentProps> = ({
```

Reply:
Yeah, I am a bit split on this. I started using `FunctionComponent`, but went for `FC` due to lazyness. Totally agree it is more explicit, though this type is not really what tells us this is a component. It is the folder name, file name and not at least the JSX in the file. We would never like... "what is this thing? Oh, let me check the type... hm React.FC... hmmmm... oh, it is a component", you know long before you look at the type that it is a component 😄